azuredeploy.parameters.region.json

```diff
@@ -0,0 +1,37 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+      "location": {
+        "value": "<resource-location>"
+      },
+      "targetVnetResourceId": {
+        "value": "<cluster-spoke-vnet-resource-id>"
+      },
+      "appInstanceId": {
+        "value": "<app-instance-id>"
+      },
+      "clusterAdminAadGroupObjectId": {
+        "value": "<azure-ad-aks-admin-group-object-id>"
+      },
+      "k8sControlPlaneAuthorizationTenantId": {
+        "value": "<tenant-id-with-user-admin-permissions>"
+      },
+      "clusterInternalLoadBalancerIpAddress": {
+        "value": "<cluster-internal-load-balancer-ip-address>"
```

Comment:
please let's complete in here and other place with already known information.

Reply:
azuredeploy.parameters.region.json this is the generic template, it will generate azuredeploy.parameters.region1.json and azuredeploy.parameters.region2.json depending the transformation. Each one has his own value, so from the template point of view is a parameter.
  #Region1
        cat ./**azuredeploy.parameters.region.json** | \
        sed "s#<resource-location>#eastus2#g" | \
        sed "s#<geo-redundancy-location>#centralus#g" | \
        sed "s#<cluster-spoke-vnet-resource-id>#${RESOURCEID_VNET_BU0001A0042_03}#g" | \
        sed "s#<tenant-id-with-user-admin-permissions>#${TENANTID_K8SRBAC}#g" | \
        sed "s#<azure-ad-aks-admin-group-object-id>#${AADOBJECTID_GROUP_CLUSTERADMIN}#g" | \
        sed "s#<cluster-internal-load-balancer-ip-address>#10.243.4.4#g" | \
        sed "s#<app-instance-id>#03#g" | \
        sed "s#<log-analytics-workspace-id>#${LOGANALYTICSWORKSPACEID}#g" | \
        sed "s#<container-registry-id>#${CONTAINERREGISTRYID}#g" | \
        sed "s#<acrPrivateDns-zones-id>#${ACRPRIVATEDNSZONESID}#g" | \
        sed "s#<subdomain-name>#${CLUSTER_SUBDOMAIN_03}#g" \
        > **azuredeploy.parameters.region1.json**

        #Region2
        cat ./**azuredeploy.parameters.region.json** | \
        sed "s#<resource-location>#centralus#g" | \
        sed "s#<geo-redundancy-location>#centralus#g" | \
        sed "s#<cluster-spoke-vnet-resource-id>#${RESOURCEID_VNET_BU0001A0042_04}#g" | \
        sed "s#<tenant-id-with-user-admin-permissions>#${TENANTID_K8SRBAC}#g" | \
        sed "s#<azure-ad-aks-admin-group-object-id>#${AADOBJECTID_GROUP_CLUSTERADMIN}#g" | \
        sed "s#<cluster-internal-load-balancer-ip-address>#10.244.4.4#g" | \
        sed "s#<app-instance-id>#04#g" | \
        sed "s#<log-analytics-workspace-id>#${LOGANALYTICSWORKSPACEID}#g" | \
        sed "s#<container-registry-id>#${CONTAINERREGISTRYID}#g" | \
        sed "s#<acrPrivateDns-zones-id>#${ACRPRIVATEDNSZONESID}#g" | \
        sed "s#<subdomain-name>#${CLUSTER_SUBDOMAIN_04}#g" \
        > **azuredeploy.parameters.region2.json**